CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [7.0.0](https://github.com/theforeman/puppet-qpid/tree/7.0.0) (2021-01-25)
+
+[Full Changelog](https://github.com/theforeman/puppet-qpid/compare/6.2.0...7.0.0)
+
+**Implemented enhancements:**
+
+- Fixes [\#30252](https://projects.theforeman.org/issues/30252): Make wait for port more resilient [\#137](https://github.com/theforeman/puppet-qpid/pull/137) ([ehelms](https://github.com/ehelms))
+
+## [6.2.0](https://github.com/theforeman/puppet-qpid/tree/6.2.0) (2020-11-30)
+
+[Full Changelog](https://github.com/theforeman/puppet-qpid/compare/6.1.0...6.2.0)
+
+**Breaking changes:**
+
+- Use modern facts [\#134](https://github.com/theforeman/puppet-qpid/issues/134)
```

Comment:
This doesn't make sense and I don't think this matches reality. 6.2 should not have a breaking change.